File: Sololearn CodeCoach Challenges/Python/PasswordValidation.py
string = input()
num_count = 0
char_count = 0
char  = ["!","@","#","$","%","&","*"]
nums  = ["1","2","3","4","5","6","7","8","9","0"]

for var in string:
    if var in char:
        char_count  += 1
    if var in nums:
        num_count  += 1
        

if (char_count > 1 and num_count > 1) and (len(string) > 6):
    print("Strong")
        
else:
    print("Weak")

File: Sololearn CodeCoach Challenges/C#/PasswordValidation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            string password = Console.ReadLine();
            int numcount = 0;
            int charcount = 0;
            string nums = "0123456789";
            string specials = "@#$!%*&";
            
            foreach (char c in password)
            {
              foreach (char sp in specials)
              {
                  if (sp == c)
                  {
                    charcount++;
                  }
              }   
            
              foreach (char num in nums)
              {
                  if (c == num)
                  {
                      numcount++;
                  }
              }
            }
            
            if (numcount >=2 && charcount >= 2 && password.Length >= 7)
            {
                Console.WriteLine("Strong");
            } 
            else 
            {
                Console.WriteLine("Weak");
            }
        }
    }
}
